src/applications/vaos/tests/new-appointment/components/ReviewPage/index.direct.unit.spec.jsx

```diff
@@ -169,8 +170,9 @@ describe('VAOS <ReviewPage> direct scheduling', () => {
 
   it('should submit successfully', async () => {
     mockAppointmentSubmit({});
+    mockPreferences('test@va.gov');
 
-    const screen = renderWithStoreAndRouter(<Route component={ReviewPage} />, {
```

Comment:
This was an old pattern that we don't need to do anymore, so I've cleaned up a few where I encountered them.